### PR TITLE
Added PlaceholderBuilder

### PR DESCRIPTION
**Pull request by PeachesMLG**
Added PlaceholderBuilder to provide more continuity between placeholders

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaceholderBuilder {
+    private final List<Placeholder> placeholderList = new ArrayList<>();
+
+    public PlaceholderBuilder() {
+        placeholderList.add(new Placeholder("prefix", IridiumSkyblock.getInstance().getConfiguration().prefix));
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java:16]: You should probably write a method to do this... Much duplicate code at the moment

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java (hunk 1 above)
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java:16]: a method to do what? I am confusion

Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java:16]: To register a placeholder replacement

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java:16]: No cuz if its an object we can parse it to ItemStacks and other sutff, anyways ima merge this. if u think its shit do a pr
:heart:

**Pull request merged**